### PR TITLE
ENH: Closes #19: return topography with undefined (masked) data filled

### LINKED ISSUE #19: topography method returning a Topography with filled masked values

### PR DESCRIPTION
**Pull request by AntoineSIMTEK**

### CODE REVIEW

File: SurfaceTopography/Uniform/common.py [hunk 1]
```diff
@@ -225,10 +225,48 @@ def plot(topography, subplot_location=111):
...
+    ----------
+    topography: Topography or UniformLineScan instance
+    fill_value: float or array of floats
+        masked value in topography will be replaced
+    """
+    heights = topography.heights()
+    heights = np.ma.filled(heights, fill_value=fill_value)
+
+    if topography.dim == 1:
+        t = UniformLineScan(heights,
```
Comment (pastewka) [inline SurfaceTopography/Uniform/common.py:246]: Is this what we want? This is breaking the pipeline concept, as it is returning a new topography.

Comment (AntoineSIMTEK) [reply SurfaceTopography/Uniform/common.py:246]: I agree, but since the application is contact mechanics, we would make a `squeeze` afterwards anyway

Comment (pastewka) [reply SurfaceTopography/Uniform/common.py:246]: Still that should be at the discretion of the user. Please implement a proper pipeline class that handles this.

Comment (AntoineSIMTEK) [reply SurfaceTopography/Uniform/common.py:246]: Ok

**Comment by AntoineSIMTEK** [comment]
It was actually easier to implement this way. Thank you.

**Comment by AntoineSIMTEK** [comment]
Sorry. I should ask for review only when the tests are passed.

**Comment by AntoineSIMTEK** [comment]
@mcrot, as discussed, I have included the changelog for your last pull-request

**Comment by AntoineSIMTEK** [comment]
Yes and PEP8
So I merge now ?

**Comment by mcrot** [comment]
I could resolve the changelog conflict directly here, therefore the new build and review request. 
But you can merge if you want, yes! Thanks.

**Pull request merged**